src/Compilers/CSharp/Portable/Binder/Binder_Constraints.cs

```diff
@@ -223,13 +223,12 @@ internal partial class Binder
                             var typeSyntax = typeConstraintSyntax.Type;
                             var typeSyntaxKind = typeSyntax.Kind();
 
-                            // For pointer types, don't report this error. It is already reported during binding typeSyntax below.
```

Comment:
Was this a regression?

Reply:
Compared to the native compiler? Yes. Compared to initial versions of roslyn? I'm not certain.

---
In reply to: [354631451](https://github.com/dotnet/roslyn/pull/40195#discussion_r354631451) [](ancestors = 354631451)